internal/handlers/telegram/handler.go

```diff
@@ -47,9 +51,12 @@ messageHandler handles the Message Telegram Object, which formats the
 Telegram update into a simple string for IRC.
 */
 func messageHandler(tg *Client, u tgbotapi.Update) {
-	formatted := tg.Settings.Prefix + u.Message.From.UserName +
-		tg.Settings.Suffix + u.Message.Text
-
+	formatted := fmt.Sprintf("%s%s%s %s",
+		tg.Settings.Prefix,
+		GetUsername(u.Message.From),
```

Comment:
So this is actually a bit of a misnomer. I thought about changing the name of this function to `GetUsernameFull` or something to that extent. This is just a helper function to properly format a person's name and username into
```
test (@testUser)
```
What _you_ want to do here is call `u.Message.From.String()` which will properly grab the `UserName` if exists, or `FirstName` field otherwise.

Reply:
alright Ill update it

Reply:
fixed